components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -904,7 +905,14 @@ void AdsServiceImpl::SetIdleThreshold(const int threshold) {
 
 bool AdsServiceImpl::IsNotificationsAvailable() const {
 #if BUILDFLAG(ENABLE_NATIVE_NOTIFICATIONS)
+
+#if defined(OS_ANDROID)
```

Comment:
We do not need the `if defined` as NotificationHelper will fall back to the default implementation which should return `true`